build.gradle

```diff
@@ -151,4 +154,15 @@ ext {
     workmanager = [
             runtimektx: "androidx.work:work-runtime-ktx:${versions.workmanager}"
     ]
+
+    material = [
+            material: "com.google.android.material:material:${versions.material}"
+    ]
+
+    androidx = [
+            appcompat       : "androidx.appcompat:appcompat:${versions.appcompat}",
```

Comment:
We already have appcompat and constraintlayout, above.

Reply:
Done